pytest_fixtures/component/provision_pxe.py

```diff
@@ -47,18 +52,19 @@ def module_provisioning_rhel_content(
     content_view = sat.api.ContentView(organization=module_sca_manifest_org).create()
 
     # Custom Content for Client repo
-    custom_product = sat.api.Product(
-        organization=module_sca_manifest_org, name=f'rhel{rhel_ver}_{gen_string("alpha")}'
-    ).create()
-    client_repo = sat.api.Repository(
-        organization=module_sca_manifest_org,
-        product=custom_product,
-        content_type='yum',
-        url=settings.repos.SATCLIENT_REPO[f'rhel{rhel_ver}'],
-    ).create()
-    task = client_repo.sync(synchronous=False)
-    tasks.append(task)
-    content_view.repository = [client_repo]
+    if is_open('SAT-27193'):
```

Comment:
This could skip client repo sync for all versions, 
```suggestion
    if is_open('SAT-27193') and int(rhel_ver) == 10:
```

Reply:
My bad, it should sync when `int(rhel_ver) != 10:`